versioned/version0_3_2.ts

```diff
@@ -68,13 +68,10 @@ class Version0_3_2 extends Versioned {
             case "contract": {
               const p = new Parser();
               const michelsonCode = p.parseMichelineExpression(x.values.lambda);
-              let meta = !!x.values.metadata
-                ? convert(x.values.metadata)
-                : null;
 
               return {
                 execute_lambda: {
-                  metadata: meta,
+                  metadata: null,
```

Comment:
Just to make sure, is it still working well on the history without metadata?

Reply:
yes, it works.